docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -241,28 +241,33 @@ results in:
  200 | b,c
 ```
 
-This aggregation function can be also used with the `FILTER` keyword to specify
-which rows are processed during the `listagg` aggregation:
+This aggregation function supports 
+{ref}`filtering during aggregation <aggregate-function-filtering-during-aggregation>`:
 
-```sql
-SELECT listagg(value, ',')
-    WITHIN GROUP (ORDER BY id)
-    FILTER (WHERE id % 2 = 0) csv_value
-FROM (VALUES
+```
+SELECT 
+    id,
+    listagg(value, ',')
+        WITHIN GROUP (ORDER BY id)
+        FILTER (WHERE id % 2 = 0) csv_value
+FROM (VALUES 
     (1, 'a'),
     (2, 'b'),
     (3, 'c'),
-    (4, 'd')
+    (2, 'd')
 ) t(id, value)
+GROUP BY id
+ORDER BY id;
 ```
 
-The example aggregates rows that have even-numbered `id`, and concatenates
-`value` to a comma-separated string:
+results in:
 
-```
- csv_value
------------
- b,d
+```text
+ id | csv_value 
+----+-----------
+  1 | NULL      
+  2 | b,d       
+  3 | NULL
 ```
 
```

Comment:
We add why there are null values for 1 and 3 even though there are values in the source table .. reason is that odd if values are filtered out .. it only obvious if you already understand what the filter does but readers should be informed explicitly